### DISCUSSION TITLE
Is it possible to upload an image in SVG format?

### DISCUSSION THREAD

**Discussion by MarcoAppleby** [globaleaks/globaleaks-whistleblowing-software#4552] [Q&A] [answered]
Hi,

I've tried to upload a logo in SVG format but the platform can't view it correctly.

Is possible create a compatibility with svg format?

Thanks

**Comment by evilaliv3** [comment, marked as answer]
Dear Marco,

Thank you for your question, we will consider your feedback for further
development.

Currently globaleaks accepts only images in PNG format.
We prefer this format to have limited codebase and exposure to file types
that requires the browser to process them. In addition browsers support for
SVG files vary so the result of loading of an SVG could be different
depending on the browser used.

**Comment by evilaliv3** [comment]
Dear @MarcoAppleby , i confirm that folowing our last security audit we have evaluated that we will not support SVG images for a matter of security. Thank you anyhow for your valuable feedback
